src/app/Console/Commands/Install.php

```diff
@@ -77,6 +77,11 @@ public function handle()
         $process->run();
         $this->closeProgressBlock();
 
+        // Install Backpack Basset
+        $this->progressBlock('Installing Basset');
+        $this->executeArtisanProcess('basset:install');
```

Comment:
When testing this, I discovered when I run `php artisan backpack:install` on a fresh project, the storage link gets added but the process hangs after that (before adding to `composer.json`). I suspected it was because the command asks the question of whether you want to do that. 

In this case (when installing Backpack), I think it's perfectly fine to assume they want it:

```suggestion
        $this->executeArtisanProcess('basset:install --no-interaction');
```